doc/semitrans.xml

```diff
@@ -12,18 +12,20 @@
 <ManSection>
   <Attr Name = "GeneratorsSmallest" Arg = "S" Label = "for a transformation
     semigroup"/>
-  <Returns>A generating set of transformations.</Returns>
+  <Returns>The smallest generating set of <A>S</A>.</Returns>
   <Description>
-    <C>GeneratorsSmallest</C> returns the lexicographically least collection
-    <C>X</C> of transformations such that <A>S</A> is generated by <C>X</C> and
-    each <C>X[i]</C> is not generated by <C>X[1], X[2], ..., X[i-1]</C>. <P/>
+    For a transformation semigroup <A>S</A>, <C>GeneratorsSmallest</C> returns
+    the   lexicographically least set of elements <C>X</C> such that <C>X</C>
+    generates <A>S</A> as a semigroup. The set <C>X</C> is lexicographically
```

Comment:
In order to be mathematically rigorous, the full stop after "semigroup" should be replaced by ", and such that"